locales/en.json

```diff
@@ -37,18 +37,59 @@
 				"overview": "Dashboard",
 				"tasks": "Tasks"
 			}
+		},
+		"language": {
+			"label": {
+				"de": "Deutsch",
+				"en": "English"
+			}
 		}
 	},
 	"homework": {
 		"button": {
 			"downloadUngradedFiles": "Download files of ungraded homework"
 		}
 	},
+	"lib": {
+		"form_ChangeLanguage" : {
+			"label": "Language",
```

Comment:
(same here)

Reply:
![image](https://user-images.githubusercontent.com/40116220/92077547-f055a280-edbc-11ea-9ae5-12096365140f.png)
needs to change

Reply:
done ;)